moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -47,6 +47,7 @@ namespace
 const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_servo.servo");
 constexpr double ROBOT_STATE_WAIT_TIME = 5.0;  // seconds
 constexpr double STOPPED_VELOCITY_EPS = 1e-4;
+constexpr auto ROS_LOG_THROTTLE_PERIOD = std::chrono::milliseconds(3000).count();
```

Comment:
I think the warning throttle just takes the time in seconds, so it seems roundabout to make a `chrono` duration to then just extract the count.

```suggestion
constexpr size_t ROS_LOG_THROTTLE_PERIOD = 3;  // seconds
```

Reply:
```
 * \param duration The duration of the throttle interval as an integral value in milliseconds.
```
I followed the pattern from the old implementation, it felt more expressive, I guess it can be a `size_t` with a comment also that it is "milliseconds"

Reply:
Oh wait, it accepts milliseconds, not seconds? So that means that the thing you copied from @AndyZe 's old implementation is actually the correct one, and the other stuff in the moveit2 repo is wrongly using seconds as milliseconds. Cool.

As you can see, my confusion is shared in https://github.com/ros2/rclcpp/issues/1929, which indeed would be nice.

OK you can leave it as is.